fili-core/src/test/groovy/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProviderSpec.groovy

```diff
@@ -25,50 +31,18 @@ class LuceneSearchProviderSpec extends SearchProviderSpec<LuceneSearchProvider>
     int rowLimit
     int searchTimeout
 
-    String sourceDir
-    String destinationDir
-
-    Path sourcePath
-    Path file1
-    Path file2
-    Path file3
-    Path subDir
-    Path file4
-    Path destinationPath
```

Comment:
`file1`, `file2`, `file3`, `file4`, and `Path subDir` are completely removed since they are not used anywhere in this Spec.

`source*` and `destination*` are used in only 1 test and, hence, are pushed down into that single test